scripts/pi-hole/js/queryads.js

```diff
@@ -16,7 +16,7 @@ function eventsource() {
     }
 
     var host = window.location.host;
-    var source = new EventSource("http://"+host+"/admin/scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+"&"+exact);
+    var source = new EventSource("//"+host+"/admin/scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+"&"+exact);
```

Comment:
That change causes the block page to not work, as it tries to find the script at http://<some_ad_domain>/scripts/...
```
EventSource's response has a MIME type ("text/html") that is not "text/event-stream". Aborting the connection.
```